app/controllers/users_controller.rb

```diff
@@ -3,7 +3,9 @@ def new
   end
 
   def create
-    @user = User.where(email: params[:user][:email]).first_or_initialize
+    @user = User.where(
+      email: params[:user][:email]
+    ).first_or_initialize(password: SecureRandom.uuid) # tmp pw for now
```

Comment:
I'm not following the "why" on this one. Why initialize with a random password? Is it a validation issue we're looking to get around?

Reply:
Yes, the users are required to have a password set when they are created.